pkg/agent/billing/clients.go

```diff
@@ -69,7 +70,7 @@ func createClients(ctx context.Context, logger *zap.Logger, cfg ClientsConfig) (
 			Name:            "azureblob",
 			Base:            client,
 			BaseConfig:      c.BaseClientConfig,
-			NewBatchBuilder: jsonArrayBatch(reporting.NewGZIPBuffer),
+			NewBatchBuilder: jsonLinesBatch(reporting.NewGZIPBuffer),
```

Comment:
Is there value in preserving old behaviour through adding a configuration option to select the format? 

This would be helpful for the rollout.

Reply:
We don't actually use current data from S3 anywhere. It exists only as a backup. There is no need for a configuration option, imo. No one will ever switch it back after this update.